DOC: update doctr instruction (#79)
NSLS-II/scientific-python-cookiecutter: docs/source/publishing-docs.rst

Travis-CI to automatically publish your documentation every time the master
branch is updated.

.. warning::
    The repo you want to build the docs for has to be a root repo.

>     You cannot build docs for a forked repo by doctr.

Reviewer: ```suggestion
    You cannot build docs for a forked repo by doctr yet.
```

Reviewer 2: I think you'll need a line break after `..warning::` for this to parse correctly.

>     doctr is working on enable forked repo build under `PR343 <https://github.com/drdoctr/doctr/pull/343>`_

Reviewer: ```suggestion
    The doctr team is working on enabling a forked repo build under
    `PR #343 <https://github.com/drdoctr/doctr/pull/343>`_.
```

Travis-CI to automatically publish your documentation every time the master
branch is updated.

> **WANRING: The repo you want to build the docs for has to be a root repo. You

Reviewer: ```suggestion
**WARNING: The repo you want to build the docs for has to be a root repo. You
```
